server/src/main/java/org/opensearch/tasks/CancellableTask.java

```diff
@@ -50,6 +50,14 @@ public abstract class CancellableTask extends Task {
     private volatile String reason;
     private final AtomicBoolean cancelled = new AtomicBoolean(false);
     private final TimeValue cancelAfterTimeInterval;
+    /**
+     * The task's cancellation start time as a wall clock time since epoch ({@link System#currentTimeMillis()} style).
+     */
+    private long cancellationStartTime = -1;
```

Comment:
This is a confusing name because it could mean "time when cancellation started" vs. "time when the task was cancelled". So which one is it? I think you mean `cancelledAt`?

Reply:
I will change in case it looks confusing. It basically means the time when task was cancelled. I will change this to ``cancelledAt`` if it looks okay to you?